src/main/java/com/nylas/Event.java

```diff
@@ -309,6 +349,112 @@ public String toString() {
 		}
 	}
 
+	public static class Notification {
```

Comment:
I think it would be better to use an abstract Notification superclass with subtypes for Email, SMS, and WebHook's, similar to the Event.When subtypes.

Reply:
Good catch, added that in now